### ISSUE TITLE
VoiceOver: The system's "go back" gesture does not work on the server selection screen

### ISSUE THREAD

**Issue by sevapopov2** [AdguardTeam/AdGuardVPNForiOS#472]
### Please answer the following questions for yourself before submitting an issue.

- [x] I checked the [knowledge base](https://adguard-vpn.com/kb/) and found no answer
- [x] I checked to make sure that this issue has not already been filed

### AdGuard VPN  version

2.9.1

### Environment

- OS: 26.1.
- Device: iPhone 13.

### Operating mode

VPN

### Exclusions mode

None, AdGuard VPN app works in integrated mode

### Which DNS server do you use?

None, AdGuard VPN app works in integrated mode

### Issue Details

Precondition:
VoiceOver is running.

Steps to reproduce:
1. Launch AdGuard-VPN
2. Press on an element with server location.
3. Perform VoiceOver's two fingersscrub gesture to go back
4. Search for a button to close the screen and go back without connecting to any server

### Expected Behavior

There is a way to go back to the main screen with screenreader enabled.

### Actual Behavior

There is no way to go back to the main screen without connecting to the server with screenreader enabled.

### Screenshots

<details><summary>Screenshot 1</summary>

</details>

### Additional Information

The issue also appears on settings screen (E.G. Open settings tab, press on general element and try to go back out of the screen. The only way to be able to use the app after going to settings screen is to restart it from app switcher and use it).

**Comment by oksenina** [comment]
@sevapopov2 
Thank you for your report! We are already working on fixing the issue and will implement the fix in future releases

**Comment by sevapopov2** [comment]
> [@sevapopov2](https://github.com/sevapopov2) Thank you for your report! We are already working on fixing the issue and will implement the fix in future releases

Hello,
The issue is now stated as closed however it is not fixed.
Should it be reopened in that case so developers remember and fix it?
Thank you!
